code/modules/client/client defines.dm

```diff
@@ -60,6 +60,7 @@
 	var/karma = 0
 	var/karma_spent = 0
 	var/karma_tab = 0
+	var/wants_karma = TRUE // let's just keep it true as default for obvious reasons
```

Comment:
Why have this at all? You could just check for the pref when someone awards the karma, and if they have the pref, then it doesn't get awarded. This is kinda a redundant var.

Reply:
its used for the proc can_get_karma which is inside karma.dm

Reply:
I know, but why? Why have a separate var when you can check for the preference bitflag instead?